commands/server/start.go

```diff
@@ -13,31 +13,34 @@ import (
 )
 
 const (
-	flagBind = "bind"
+	flagBind  = "bind"
+	flagDebug = "debug"
 )
 
-func parseBind(args []string) (string, error) {
+func parseFlags(args []string) (string, bool, error) {
 	// parse flagBind and return the result
 	var addr string
+	var debug bool
 	startFlags := flag.NewFlagSet("start", flag.ExitOnError)
 	startFlags.StringVar(&addr, flagBind, "tcp://localhost:46658", "address server listens on")
+	startFlags.BoolVar(&debug, flagDebug, false, "call stack returned with Tx result")
```

Comment:
maybe `call stack returned on Tx error` (definitely not on result, which implies success)